dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -119,7 +126,7 @@ public static String getResourceViewSuffixes() {
      * @throws IOException errors
      */
     public static void createWorkDirIfAbsent(String execLocalPath) throws IOException {
-        //if work dir exists, first delete
+        // if work dir exists, first delete
```

Comment:
```work dir``` is puzzling ，maybe we can replace that with ```work catalog``` or ```work directory```? @EricGao888 PTAL

Reply:
I'd rather we just remove this line of comment. The code has indicated what it is doing here. BTW, this comment has already been away from the code, which also shows it is hard to maintain such kind of comment.

```java
 if (execLocalPathFile.exists()) {
            try {
                org.apache.commons.io.FileUtils.forceDelete(execLocalPathFile);
......
```